src/api/lib/tasks/dev.rake

```diff
@@ -398,6 +398,9 @@ namespace :dev do
       # Trigger package builds for home:admin
       home_admin.store
 
+      # The password of every user will be 'opensuse', so we can easily log in as one of them if needed
+      User.all_without_nobody.each { |user| user.update(password: 'opensuse', password_confirmation: 'opensuse') }
+
```

Comment:
This is controlled in the [user factory](https://github.com/openSUSE/open-build-service/blob/ee129e3e08909ace74beaeed31e8d144840ffccb/src/api/spec/factories/users.rb#L6). Maybe you want to change that line. Is there any other use case for modifying the passwords with the dev rake task?

Reply:
You have a good point, this would be simpler.

> Is there any other use case for modifying the passwords with the dev rake task?

No, I didn't think about changing the factory directly.